### ISSUE TITLE
Unknow status 7

### ISSUE THREAD

**Issue by wmollenvanger** [starkillerOG/motion-blinds#19]
Got:
2022-02-22 17:27:49 ERROR (SyncWorker_4) [motionblinds.motion_blinds] Device with mac 'e868e7c229f00002' has status '7' that is not yet known, please submit an issue at https://github.com/starkillerOG/motion-blinds/issues.

These are Brel shades. 

Not sure what status is read but here you can find the API description: https://github.com/alexbacchin/ConnectorBridge/blob/master/doc/Wi-Fi%20Local%20Access/Connector%20WLAN%20Integration%20Protocol.pdf

**Comment by starkillerOG** [comment]
@wmollenvanger thank you for reporting the issue.
The status 7 is not documented in the API (also not the one you provided).
I have asked the manufacturer what status 7 means.

In the mean time you can just ignore the error, it schould not effect the operation of the HomeAssistant integration.

**Comment by starkillerOG** [comment]
I fixed the status 7 issue in the 0.6.2 release of motionblinds.
It will be fixed once this PR is merged into HomeAssistant: https://github.com/home-assistant/core/pull/68570
